pymeasure/instruments/ni/virtualbench.py

```diff
@@ -0,0 +1,1713 @@
+#
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2019 PyMeasure Developers
+# pyvirtualbench library: Copyright (c) 2015 Charles Armstrap <charles@armstrap.org>
```

Comment:
I included the copyright of the underlying library, since I copied some docstrings

Reply:
Is the underlying library licensed with the MIT License?

Reply:
yes, it's also MIT License: https://github.com/armstrap/armstrap-pyvirtualbench/blob/master/LICENSE